src/routes/vocabulary-list/VocabularyListScreen.tsx

```diff
@@ -64,16 +58,22 @@ const VocabularyListScreen = ({ route }: VocabularyListScreenPropsType): JSX.Ele
           setSelectedDocumentIndex={setSelectedDocumentIndex}
         />
       )}
+      <Title
+        title={labels.exercises.vocabularyList.title}
+        description={`${documents?.length ?? '0'} ${documents?.length === 1 ? labels.home.word : labels.home.words}`}
+      />
+
       <Loading isLoading={loading}>
-        <StyledList
-          data={documents}
-          ListHeaderComponent={Header}
-          renderItem={renderItem}
-          keyExtractor={item => `${item.id}`}
-          showsVerticalScrollIndicator={false}
-        />
+        <>
```

Comment:
Can't you change the type of `children` in `Loading` component to an array so you don't have to use fragments all the time here? Would be nicer imo

Reply:
Changed it to ReactNode-Type